apps/native-component-list/src/screens/Contacts/ContactsScreen.tsx

```diff
@@ -74,19 +59,17 @@ function ContactsView({ navigation }: { navigation: any /* StackNavigationProp<S
     const { data: nextContacts } = payload;
 
     if (restart) {
-      _rawContacts = {};
+      rawContacts = {};
     }
 
     for (const contact of nextContacts) {
-      _rawContacts[contact.id] = contact;
+      rawContacts[contact.id] = contact;
     }
-    setContacts(Object.values(_rawContacts));
-    setHasPreviousPage(payload.hasPreviousPage);
```

Comment:
is it safe to just remove this? seems like it'd break a feature

Reply:
We weren't using the result of it.